cmd/kubelet/app/server.go

```diff
@@ -474,6 +476,30 @@ func makeEventRecorder(kubeDeps *kubelet.Dependencies, nodeName types.NodeName)
 	}
 }
 
+func getReservedCPUs(machineInfo *cadvisorapi.MachineInfo, cpus string) cpuset.CPUSet {
+	emptyCPUSet := cpuset.NewCPUSet()
+	if cpus != "" {
+		topo, err := topology.Discover(machineInfo)
+		if err != nil {
+			klog.Error("Unable to discover CPU topology info")
+			return emptyCPUSet
+		}
+		reservedCPUSet, err := cpuset.Parse(cpus)
+		if err != nil {
+			klog.Error("Unable to parse reserved-cpus list")
+			return emptyCPUSet
+		}
+		allCPUSet := topo.CPUDetails.CPUs()
+		klog.Infof("Online CPUs: %s", allCPUSet.String())
```

Comment:
Do we need to always log this on each start up?

Reply:
if we failed hard as mentioned above, we don't; otherwise we do.